### PR TITLE
Artillery Sidewinder X2

### PR DESCRIPTION
**Pull request by GP3D84**

### CODE REVIEW

File: resources/profiles/Artillery.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        {
+            "name": "Artillery Generic PLA-CF",
+            "sub_path": "filament/Artillery Generic PLA-CF.json"
+        },
+        {
+            "name": "Artillery Generic PETG",
+            "sub_path": "filament/Artillery Generic PETG.json"
+        },
+        {
+            "name": "Anker Generic ABS",
```
Comment (SoftFever) [inline resources/profiles/Artillery.json:60]: Could you rename it to Artillery?

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by SoftFever** [comment]
Thank you for the PR.
Could you take a look at my comment?

File: resources/profiles/Artillery.json (hunk 1 above)
Comment (GP3D84) [reply resources/profiles/Artillery.json]: Fixed. Thanks

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
Merged.
Thank you.

**Pull request merged**